numbers.js

```diff
@@ -48,7 +46,7 @@ function enable(){
   resetButton.disabled=false;
 }
 
-disable();
+// disable();
```

Comment:
I'm unclear why you are calling the disable function here? Try it without and reassess if you need to be doing this.